specs/darwin/apps.table

```diff
@@ -2,7 +2,7 @@ table_name("apps")
 description("macOS applications installed in known search paths (e.g., /Applications).")
 schema([
     Column("name", TEXT, "Name of the Name.app folder"),
-    Column("path", TEXT, "Absolute and full Name.app path", index=True),
+    Column("path", TEXT, "Absolute and full Name.app path", index=True, optimized=True),
```

Comment:
The whole `Application path accumulator` could be less nested, but I think this is correct.